### ISSUE TITLE
Comments on Conclusion

### ISSUE THREAD

**Issue by MPillas** [MPillas/S240422ed#8]

**Comment by jgducoin** [comment]
You say "Therefore, there is a necessity to image the first moment"

"the first moment" should be rephrased as "as quickly as possible" or "promptly after the merger time", something like that.

**Comment by santier14** [comment]
@wbcorradi @MPillas yes the conclusion is a bit long, but we tried to super summarize key ideas in abstract :p
